package.json

```diff
@@ -41,6 +41,7 @@
       "node_modules/open/xdg-open",
       "node_modules/windosu/*.bat",
       "node_modules/windosu/*.cmd",
+      "npm-shrinkwrap.json",
```

Comment:
`npm-shrinkwrap.json` is now added to the `pkg` executable so that its timestamp can be used for the `cacheKiller` option of `fast-boot2`. It should be less than 200KB when compressed. (We could be better, use an empty file with a similar timestamp, hardcode exceptions... But probably not worth it.)